microsoft-edge/extensions-chromium/developer-guide/sidebar.md

```diff
@@ -0,0 +1,625 @@
+  .catch((error) => console.error(error));
+...
+```
+
+
+<!-- ------------------------------ -->
+#### Switch to a different sidebar
+
+<!-- compare https://developer.chrome.com/docs/extensions/reference/sidePanel/#multi-panels -->
+
+An extension can use [sidepanel.getOptions()](https://developer.chrome.com/docs/extensions/reference/sidePanel/#method-getOptions) to retrieve the current sidebar, and then enable a different sidebar for a specific tab.
+
+This example sets a sidebar containing a welcome message on [runtime.onInstalled()](https://developer.chrome.com/docs/extensions/reference/runtime/#event-onInstalled).  When the user navigates to a different tab, the sidebar is replaced with the browser-level sidebar.
+
+`service-worker.js`:
+
+```js
+const welcomePage = 'sidebar/welcome-sp.html';
+const mainPage = 'sidebar/main-sp.html';
+
+chrome.runtime.onInstalled.addListener(() => {
+  chrome.sidePanel.setOptions({ path: welcomePage });
+});
+
+chrome.tabs.onActivated.addListener(async ({ tabId }) => {
+  const { path } = await chrome.sidePanel.getOptions({ tabId });
+  if (path === welcomePage) {
+    chrome.sidePanel.setOptions({ path: mainPage });
+  }
+});
+```
+
+
+<!-- ====================================================================== -->
+## The sidebar user experience (UX)
+
+<!-- compare https://developer.chrome.com/docs/extensions/reference/sidePanel/#user-experience -->
+
+Extensions in the Microsoft Edge sidebar have these user experience (UX) features.
+
+
+<!-- ------------------------------ -->
+#### How extensions that have sidebar support are indicated
+
+* The sidebar icon is displayed next to the extension name in the hub in the toolbar:
+
+![Sidebar dialog](./sidebar-images/sidebar-dialog.png)
+
+
+###### Using the action toolbar icon
+
+Users can open the sidebar by clicking the extension shortcut in the toolbar, if it's enabled.
+
+See [Enable the extension's shortcut icon to open the sidebar](#enable-the-extensions-shortcut-icon-to-open-the-sidebar), above.
+
+
+<!-- todo: un-comment & revise, after implemented:
+###### Using a keyboard shortcut
+
+Users can open the sidebar by pressing a keyboard shortcut, if the [action command](https://developer.chrome.com/docs/extensions/reference/commands/#action-commands) and the [action icon](https://developer.chrome.com/docs/extensions/reference/sidePanel/#open-action-icon) are enabled.
+-->
+
+
+<!-- ------------------------------ -->
+#### Opening the sidebar
+
+<!-- compare https://developer.chrome.com/docs/extensions/reference/sidePanel/#open -->
+
+To open a sidebar extension, the user can do either of the following:
+
+* Click the sidebar icon in the hub in the toolbar.
+
+* Right-click the extension's shortcut in the toolbar.
+
+<!-- true but maybe not needed to point out:
+* On the Microsoft Edge Add-Ons store page, there's a information message on the extension product page, highlighting the sidebar feature. -->
```

Comment:
Seems like this would be good to have under the `How extensions that have sidebar support are indicated` section?

Reply:
todo: move sentence to "How extensions that have sidebar support are indicated"

Reply:
restructured UX section locally, it will need re-review